zwave_js_server/model/node/__init__.py

```diff
@@ -138,7 +138,7 @@ def index(self) -> int:
     @property
     def device_class(self) -> DeviceClass:
```

Comment:
Maybe this should return None instead and we don't change the properties of the `DeviceClass` class?